### PR TITLE
Update syntax for Swift 3

### PR DESCRIPTION
**Pull request by Elland**

### CODE REVIEW

File: Sources/NSFileManager+Helpers.swift [hunk 1]
```diff
@@ -2,14 +2,12 @@ import Foundation
 
 extension FileManager {
     public func exists(at url: URL) -> Bool {
-        guard let path = url.path else { fatalError("Couldn't get path for url: \(url)") }
-
+        let path = url.path
```
Comment (3lvis) [inline Sources/NSFileManager+Helpers.swift]: New line before return, please

File: Sources/Networking.swift [hunk 2]
```diff
@@ -120,7 +120,7 @@ public class Networking {
     var token: String?
     var authorizationHeaderValue: String?
     var authorizationHeaderKey = "Authorization"
-    var cache: Cache<AnyObject, AnyObject>
+    var cache: NSCache<AnyObject, AnyObject>
```
Comment (3lvis) [inline Sources/Networking.swift]: lol

File: Sources/Networking.swift [hunk 3]
```diff
@@ -319,21 +319,21 @@ extension Networking {
...
 
-        if let object = self.cache.object(forKey: destinationURL.absoluteString!) {
+        if let object = self.cache.object(forKey: destinationURL.absoluteString) {
             completion(object: object)
         } else if FileManager.default.exists(at: destinationURL) {
             let semaphore = DispatchSemaphore(value: 0)
             var returnedObject: AnyObject?
 
-            DispatchQueue.global(attributes: DispatchQueue.GlobalAttributes.qosUtility).async {
+            DispatchQueue.global(qos: .utility).async {
```
Comment (3lvis) [inline Sources/Networking.swift]: qos... ewww

Comment (Elland) [inline Sources/Networking.swift]: two steps forward, one step back. 😢

**Pull request merged**